### PR TITLE
feat(quantiy): change padding to allow 4 digit

### PR DESCRIPTION
**Pull request by EnzoStll**

### CODE REVIEW

File: packages/stencil/components/quantity/src/index.html [hunk 1]
```diff
@@ -20,15 +20,15 @@
 <p>Quantity using osds-components</p>
 <osds-quantity onclick="globalMethod()" id="quantity-1">
   <osds-button slot="minus" color="primary" size="sm">-</osds-button>
-  <osds-input style="max-width: 45px" type="number" color="primary" min="1" max="10" step="1
+  <osds-input style="max-width: 45px; padding: 0%" type="number" color="primary" min="1" max="9999" step="1
```
Comment (skhamvon) [inline packages/stencil/components/quantity/src/index.html:23]: Peux tu enlever le padding inline ?
Pareil pour la ligne 31

**Comment by ovh-cds** [comment]
CDS Report build-storybook#539.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#540.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#540.1 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#540.2 ✘  
* 
  * build ✘

**Review by astagnol — CHANGES_REQUESTED** [review]
Nice! But you should update the following things:
- in osds-quantity.e2e.screenshot.ts: add style="width: 45px" to <osds-input> and <input> so screenshot tests will be similar to expected integration (up to 4 digits input)
- in index.html: modify max-width to width since integrator should set the width of the specific input and not define the maximum size the input element is allow to have
- for e2e tests, I'll call you :)

**Pull request closed without merging**